gtfs-realtime/CHANGES.md

```diff
@@ -117,4 +117,5 @@ message MyTripDescriptorExtension {
 |1008|SEPTA - Southeastern Pennsylvania Transportation Authority|[Gregory Apessos](mailto:GApessos@septa.org)|https://github.com/septadev|
 |1009|Swiftly|[mike@goswift.ly](mailto:mike@goswift.ly)|[Group Discussion](https://groups.google.com/forum/#!msg/gtfs-realtime/mmnZV6L-2ls/wVWdknhLBwAJ)|
 |1010|IBI Group|[Ritesh Warade](mailto:transitrealtime@ibigroup.com)|[GitHub proposal for new timestamps in Service Alerts](https://github.com/google/transit/pull/134)|
-|9000-9999|RESERVED - INTERNAL USE ONLY|[GTFS Community](https://groups.google.com/forum/#!forum/gtfs-realtime)|[Group discussion](https://github.com/google/transit/pull/178/)|
+|1013|MITFAHR⏐DE⏐ZENTRALE(MFDZ)|[Holger Bruch](mailto:holger.bruch@mfdz.de)|[Group discussion](https://groups.google.com/g/gtfs-realtime/c/IxYh-beoNoo)|
```

Comment:
Heads up: This only *doesn't* mess up the Markdown table, because the 2nd column (`MITFAHR⏐DE⏐ZENTRALE(MFDZ)`) uses Unicode vertical bars, instead of ASCII ones (`|`). A more robust option might be to use ASCII bars and [escape](https://github.github.com/gfm/#backslash-escapes) them.

Reply:
Thank you @derhuerst for pointing this out! I've updated this.